Backend/Backend/Infrastructure/FileStorageHelper.cs

```diff
@@ -0,0 +1,296 @@
+using static System.IO.Directory;
+using static System.IO.Path;
+using static System.IO.File;
+
+namespace Backend.Infrastructure;
+
+/// <summary>
+/// Handles File interactions to/from the server.
+/// </summary>
+public static class FileStorageHelper
+{
+
+    /// <summary>
+    /// Seperator key for the file name and file type from the filename stored in the db.
+    /// </summary>
+    public const string FILE_SPLIT_KEY = "|/|\\|";
+
+    /// <summary>
+    /// Base dir where all server files are stored.
+    /// </summary>
+    public const string BASE_DIR = "ServerFiles";
+
+    /// <summary>
+    /// Dir where all user avatars are stored.
+    /// </summary>
+    public const string AVATARS_DIR_NAME = "Avatars";
+
+    /// <summary>
+    /// Dir where all podcast covers and episodes are stored. 
+    /// </summary>
+    public const string PODCASTS_DIR_NAME = "Podcasts";
+
+    #region User Profile
+
+    /// <summary>
```

Comment:
I understand why you did this, but its kinda redundant. why not just use a toString() in the original call?

Reply:
...same with this one. I mean to comment on the method
public static string SaveUserAvatar(Guid userId, IFormFile avatarFile)
    {
        return SaveUserAvatar(userId.ToString(), avatarFile);
    }

Reply:
For ease of use.